documentation/slate/source/includes/stuff.md

```diff
@@ -53,6 +53,61 @@ The different journey types are:
 |non_pt_bike|A journey without public transport, only biking|
 |non_pt_bss|A journey without public transport, only bike sharing|
 
+<a name=Service-translation></a>Service Translation
+-------------------------------------------
+When specifying a service in the data through `calendar.txt` and `calendar_dates.txt`, you might get surprised to see a different result from Navitia's response.
+
+```shell
+# Navitia's interpreted service response
+{
+    "calendars": [
+       {
+            "active_periods": [
+                {
+                    "begin": "20200101",
+                    "end": "20200130"
+                }
+            ],
+            "week_pattern": {
+                "monday": false,
+                "tuesday": false,
+                "wednesday": false,
+                "thursday": false,
+                "friday": false,
+                "saturday": true,
+                "sunday": true
+            },
+            "exceptions": [
+                {
+                    "type": "remove",
+                    "datetime": "20200127"
+                }
+            ]
+        }
+    ]
+```
+
+
+For instance, if you have a trip that:
+
+-   occurs every Saturday of January 2020 (in your `calendar.txt`)
+-   has 3 exceptions that `add` the service the first 3 Sundays (`calendar_dates.txt`)
+
+
+You might expect to have that same exact representation for your `/vehicle_journeys`, but instead you find something different.
+
```

Comment:
Maybe move here the code block with "Navitia's interpreted service response"

Reply:
That's what I did at first but the rendering was weird as the code gets displayed on the right hand side of the web page. 
I agree it makes more sense when you read the code, but when looking at the rendered HTML page, it's much better up there :/

Reply:
Ok !